osler/assets/inventory/drug-list/DrugListTable.js

```diff
@@ -0,0 +1,193 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import TableManager from '../../core/all-patients/TableManager';
+import CSRFToken from '../../util/CSRFToken';
+import compare from '../../util/compare';
+import simpleComparator from '../../util/simpleComparator';
+import Container from 'react-bootstrap/Container';
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Form from 'react-bootstrap/Form';
+import globalFilter from './globalFilter';
+import { DateTime } from 'luxon';
+
+
+function DrugListTable(props) {
+
+  const [state, setState] = useState({ 'show': false, 'drug': {} });
+
+  const handleClose = () => {
+    setState({ 'show': false, 'drug': {} });
+  }
+  const handleShow = (e, row) => {
+    setState({ 'show': true, 'drug': row });
+  }
+
+  const nameComparator = simpleComparator('name');
+  const categoryComparator = simpleComparator('category');
+  const expirationDateComparator = simpleComparator('expiration_date');
+  const stockComparator = simpleComparator('stock');
```

Comment:
There's still a bug with the comparators. It seems that all the comparators after nameComparator are just returning the same result as the name comparator. I'm fairly certain this is a bug related to the memoization in simpleComparator. useMemo() is required by React Table, and it allows you to specify dependencies, which I'm guessing we need to do

Reply:
Correct sorting is something that the selenium tests really need to check for